docs/ecs-compose-features.md

```diff
@@ -0,0 +1,165 @@
+# Compose - Amazon ECS mapping
+
+This document outlines the conversion of an application defined in a Compose file to AWS resources.
+Each service is mapped to an ECS service in the project's cluster.
+
+## Compose fields mapping
+
+The table below lists supported Compose file fields and their AWS counterparts.
+
+__Legend:__
+
+- __✓:__ Implemented
+- __n:__ Not yet implemented
+- __x:__ Not applicable / no available conversion
+
+| Keys                           |Map|  Notes                                                       |
+|--------------------------------|---|--------------------------------------------------------------|
+| __Service__                    | ✓ |
+| service.service.build          | x |  Ignored. No image build support on AWS.
+| service.cap_add, cap_drop      | ✓ |  Supported with [Fargate limitations](https://docs.aws.amazon.com/AmazonECS/latest/APIReference/API_KernelCapabilities.html)
+| service.command                | ✓ |  
+| service.configs                | x |
+| service.cgroup_parent          | x |
+| service.container_name         | x |  
+| service.credential_spec        | x |
+| service.deploy                 | ✓ |
+| service.deploy.endpoint_mode   | x |
+| service.deploy.mode            | x |
+| service.deploy.replicas        | ✓ |  Set service initial scale. Auto-scaling, when enabled, will make this dynamic
+| service.deploy.placement       | ✓ |  Used with EC2 support to select a machine type and AMI
+| service.deploy.update_config   | ✓ |
+| service.deploy.resources       | ✓ |  Fargate resource is selected with the lowest instance type for configured memory and cpu
+| service.deploy.restart_policy  | ✓ |  
+| service.deploy.labels          | ✓ |  
+| service.devices                | x |
+| service.depends_on             | ✓ |  Implemented using CloudFormation Depends_on
+| service.dns                    | x |
+| service.dns_search             | x |
+| service.domainname             | x |  
+| service.tmpfs                  | x |  Not supported on Fargate, see https://github.com/docker/compose-cli/issues/839
+| service.entrypoint             | ✓ | 
+| service.env_file               | ✓ |
+| service.environment            | ✓ |
+| service.expose                 | x |
+| service.extends                | ✓ |
+| service.external_links         | x |
+| service.extra_hosts            | x |
+| service.group_add              | x |
+| service.healthcheck            | ✓ | 
```

Comment:
I thought we could not support healthchecks on ECS? (cf https://docker.slack.com/archives/C0111G2RCE4/p1600870550019100). Did I misundestand this ?

Reply:
we support container healthcheck (same a docker) but not Load Balancer healthcheck
and please don't ask me why ALB don't just rely on container healthchecks that are used to report service health on ECS console :P